src/Cms/Find.php

```diff
@@ -77,9 +77,16 @@ public static function language(string $code)
 	public static function page(string $id)
 	{
 		// decode API ID encoding
-		$id = str_replace(['+', ' '], '/', $id);
+		$id    = str_replace(['+', ' '], '/', $id);
+		$kirby = App::instance();
+
+		// support looking up the site itself,
+		// as this will fail when using $kirby->page()
+		if ($id === '/' || $id === 'site://') {
```

Comment:
This feels wrong to me. Why would the page method also be able to find the site? Is this just for convenience?

Reply:
Would you rather do it in the page move dialog submit handler?